alembic/versions/0014_add_app_user_per_org_authz.py

```diff
@@ -0,0 +1,65 @@
+"""add_app_user_per_org_authz
+
+Revision ID: 0014
+Revises: 0013
+Create Date: 2023-05-03 20:44:16.197147
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0014'
+down_revision = '0013'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
To confirm the understanding, we are creating a user table that all users will be listed in. We then add permissions for that user to have access to each organisation using the organisation_admin table with the primary key being the unique user email and org id?

Reply:
yes, correct